### ISSUE TITLE
Better Documentation and Examples (Especially Query Related Documentation and Examples)

### ISSUE THREAD

**Issue by sirinath** [ambiata/icicle#641]
For an external person have a glance at this project it is difficult to say how data is to be queried and also how to get started.

**Comment by HuwCampbell** [comment]
You're absolutely right. The project status is also not very well reflected. Check out the [ambling](https://github.com/ambiata/icicle/blob/master/doc/user/ambling.md) document to get a feel for it.

The main input file format for icicle is [Zebra](https://github.com/ambiata/zebra), but psv is supported as well.
